Tests/FunctionalTests/PluginTests.swift

```diff
@@ -107,4 +107,22 @@ class PluginTests: XCTestCase {
         }
         #endif
     }
+
+    func testUseOfVendedBinaryTool() throws {
+        // Check if the host compiler supports the '-entry-point-function-name' flag.  It's not needed for this test but is needed to build any executable from a package that uses tools version 999.0.
+        try XCTSkipUnless(doesHostSwiftCompilerSupportRenamingMainSymbol(), "skipping because host compiler doesn't support '-entry-point-function-name'")
```

Comment:
intentional?

Reply:
Unfortunately yes.  Right now the 999.0 tools version enables the functionality of building executables using the new flag, as well as enabling plugins.  I'm wondering whether we should have a separate flag to enable it.  I can also change the tests to use libraries.